docs/ru/platform-indexes.md

```
@@ -0,0 +1,1691 @@
+
+#### Правило 27 — [ОРРХ | ОРНР1 +] Идентификатор (Кластерный)
+
+Условие: Всегда.
+
+Статус: ❌ Служебная таблица опций исключена из модели решением задачи.
+
+---
+
+## Документ
+
+#### Правило 1 — [ОРНР1 + ... +] Ссылка (Кластерный)
```

markdownlint-cli2 (0.23.2)

[warning] Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)